stellar-dotnet-sdk/Transaction.cs

```diff
@@ -37,118 +34,91 @@ private Transaction(KeyPair sourceAccount, uint fee, long sequenceNumber, Operat
 
         public TimeBounds TimeBounds { get; }
 
-        public List<DecoratedSignature> Signatures { get; }
-
-        /// <summary>
-        /// Adds a new signature ed25519PublicKey to this transaction.
-        /// </summary>
-        /// <param name="signer"> signer <see cref="KeyPair"/> object representing a signer</param>
-        public void Sign(KeyPair signer)
-        {
-            Sign(signer, Network.Current);
-        }
-
         /// <summary>
-        /// Adds a new signature ed25519PublicKey to this transaction.
+        ///     Returns signature base for the given network.
         /// </summary>
-        /// <param name="signer"> signer <see cref="KeyPair"/> object representing a signer</param>
         /// <param name="network">The network <see cref="Network"/> the transaction will be sent to.</param>
-        public void Sign(KeyPair signer, Network network)
+        /// <returns></returns>
+        public override byte[] SignatureBase(Network network)
```

Comment:
I'm not sure if this will produce the correct hash for v1 transactions. this is the implementation in the Java SDK PR https://github.com/stellar/java-stellar-sdk/pull/278/files#diff-ac19703cf3fa8051734361585adb036cR55 . Can you add tests on `Hash()` for v1 transactions with and without a muxed account as the source account?

Reply:
You're right. I didn't update it.